product/src/controllers/userController.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import { Users } from '../models/users.js';
+
+export class UserController {
+   async registerAccount(req, res) {
+        const user = req.body;
+        user.password = await bcrypt.hash(user.password, 10);
```

Comment:
I think it's a better approach to put the salt in the .env file or get it from the caller